app/views/assistance/_assistance.html.haml

```diff
@@ -1,23 +1,32 @@
 .ui.segment.assistance
   %h4.ui.header= assistance.title
   %p= assistance.institution.name
-  - if assistance.expert
+  - assistance.experts.each do |expert|
     %h5= t('.contact')
-    - contact = assistance.expert
+    - contact = expert
     %p
       = contact.full_name
       %br
       = contact.role
     - expert_email = contact.email
     - mailto_text = t('.contact_by_email')
-  - else
+
+    - if visit.visitee and expert_email
```

Comment:
Prefer using `&&` rather than `and`. More info : https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby